provider-blackduck/src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProviderDataAccessor.java

```diff
@@ -257,16 +257,16 @@ public void updateProjectAndUserData(Long providerConfigId, Map<ProviderProject,
 
         Predicate<UserView> searchFilter = userView -> StringUtils.isNotBlank(userView.getEmail());
         if (StringUtils.isNotBlank(searchTerm)) {
-            searchFilter = searchFilter.and(userView -> StringUtils.containsIgnoreCase(userView.getEmail(), searchTerm));
+            searchFilter = searchFilter
+                               .and(userView -> StringUtils.containsIgnoreCase(userView.getEmail(), searchTerm) || StringUtils.containsIgnoreCase(userView.getUserName(), searchTerm));
```

Comment:
This line is doing a lot of work. Can we use a bit more abstraction or find another way to improve its readability?

Reply:
Updated: https://github.com/blackducksoftware/blackduck-alert/pull/1656/files#r654567941